### ISSUE TITLE
Test failing (PHP7): testPopulateFunctionAbstractThrowsExceptionWithInvalidNode

### ISSUE THREAD

**Issue by asgrim** [Roave/BetterReflection#111]
Build is failing [#379.2](https://travis-ci.org/Roave/BetterReflection/jobs/77288827)
```
There was 1 failure:
1) BetterReflectionTest\Reflection\ReflectionFunctionAbstractTest::testPopulateFunctionAbstractThrowsExceptionWithInvalidNode
Failed asserting that exception of type "TypeError" matches expected exception "InvalidArgumentException".
```

Fix urgently, this made it into master when #110 merged by mistake (to fix #108).

**Comment by asgrim** [comment]
Handled in #112

**Comment by Ocramius** [comment]
Closed via #112
